.github/workflows/create_version_bump_PR.yml

```diff
@@ -0,0 +1,48 @@
+name: Create version bump PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Next Version Number (x.x.x)'
+        required: true
+      is_prerelease:
+        description: 'Is Prerelease version? (true/false)'
+        default: 'false'
+        required: true
+
+jobs:
+  bump_version:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      Version: "${{ github.event.inputs.version }}"
+      IsPrerelease: "${{ github.event.inputs.is_prerelease }}"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: "Modify build-packages"
+        run: |
+          CURRENTVERSION="$(grep -o -e RELEASE_VERSION\=\"[0-9]*\.[0-9]*\.[0-9]*  dotnet/build-packages.sh | sed 's/RELEASE_VERSION="//')"
+          VERSION="${{ github.event.inputs.version }}"
+          sed -i '' -e "s/$CURRENTVERSION/"$VERSION/g" dotnet/build-packages.sh
+          git tag "$TAGNAME"
+          git push origin "$TAGNAME"
+
+      - name: Create Pull Request
+        id: pr
+        uses: peter-evans/create-pull-request@v3.10.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: "version-bump-${{steps.versions.outputs.full_version}}"
+          commit-message: "[Version Bump] ${{steps.versions.outputs.full_version}}"
+          delete-branch: true
+          title: "[Version Bump] ${{steps.versions.outputs.full_version}}"
+          body: "Dotnet Tracer Bump"
```

Comment:
Do we want to be able to use this for _non_ Dotnet Tracer Bump reasons? If so, then we could/should take this as an input choice and change the body here. We could also include that in the `[Version Bump]` commit title, and use that to drive the tag we create in the auto-tag action?

Reply:
Yes I assume we will, but I want to do the changes incrementally. ie make it work for dotnet, then discuss with the java team.